test/integration/functional_test.go

```diff
@@ -550,16 +552,17 @@ func validateMySQL(ctx context.Context, t *testing.T, profile string) {
 		t.Fatalf("%s failed: %v", rr.Args, err)
 	}
 
+	names, err := PodWait(ctx, t, profile, "default", "app=mysql", 5*time.Minute)
+	if err != nil {
+		t.Fatalf("podwait: %v", err)
+	}
+
 	// Retry, as mysqld first comes up without users configured. Scan for names in case of a reschedule.
 	mysql := func() error {
-		names, err := PodWait(ctx, t, profile, "default", "app=mysql", 5*time.Second)
-		if err != nil {
-			return err
-		}
 		rr, err = Run(t, exec.CommandContext(ctx, "kubectl", "--context", profile, "exec", names[0], "--", "mysql", "-ppassword", "-e", "show databases;"))
 		return err
 	}
-	if err = retry.Expo(mysql, 1*time.Second, 2*time.Minute); err != nil {
+	if err = retry.Expo(mysql, 10*time.Second, 60*time.Second); err != nil {
```

Comment:
10*time.Seconds seems to be too large to start, given that it is gonna be incremented exponentitally.
maybe start with 3 ?

Reply:
Done.